__tests__/integration/one-app.spec.js

```diff
@@ -1668,3 +1705,323 @@ describe('heapdump', () => {
+    originalModuleMap = readModuleMap();
+    ({ browser } = await setUpTestRunner({ oneAppLocalPortToUse, oneAppMetricsLocalPortToUse }));
+  });
+
+  afterAll(async () => {
+    await tearDownTestRunner({ browser });
+    writeModuleMap(originalModuleMap);
+  });
+
+  test('Request: /_/status', async () => {
+    const response = await fetch(`${fetchUrl}/_/status`, {
+      ...defaultFetchOptions,
+      method: 'GET',
+      headers: {
+        origin: 'test.example.com',
+      },
+    });
+
+    expect(response.status).toBe(200);
+    expect(response.headers.raw()).toEqual({
+      connection: [
+        'close',
+      ],
+      'content-length': [
+        '2',
+      ],
+      'content-type': [
+        'text/plain; charset=utf-8',
+      ],
+      date: [
+        expect.any(String),
+      ],
+      etag: [
+        expect.any(String),
+      ],
+      'one-app-version': [
+        expect.any(String),
+      ],
+      'referrer-policy': [
+        'same-origin',
+      ],
+      'strict-transport-security': [
+        'max-age=15552000; includeSubDomains',
+      ],
+      vary: [
+        'Accept-Encoding',
+      ],
+      'x-content-type-options': [
+        'nosniff',
+      ],
+      'x-frame-options': [
+        'DENY',
+      ],
+      'x-xss-protection': [
+        '1; mode=block',
+      ],
+    });
+  });
+
+  test('Request: /_/report/security/csp-violation', async () => {
+    const response = await fetch(`${fetchUrl}/_/report/security/csp-violation`, {
+      ...defaultFetchOptions,
+      method: 'POST',
+      headers: {
+        origin: 'test.example.com',
+      },
+    });
+
+    expect(response.headers.raw()).toEqual({
+      connection: [
+        'close',
+      ],
+      'content-security-policy': [
+        expect.any(String),
+      ],
+      date: [
+        expect.any(String),
+      ],
+      'expect-ct': [
+        'max-age=0',
+      ],
+      'one-app-version': [
+        expect.any(String),
+      ],
+      'referrer-policy': [
+        'no-referrer',
+      ],
+      'strict-transport-security': [
+        'max-age=15552000; includeSubDomains',
+      ],
+      'x-content-type-options': [
+        'nosniff',
+      ],
+      'x-dns-prefetch-control': [
+        'off',
+      ],
+      'x-download-options': [
+        'noopen',
+      ],
+      'x-frame-options': [
+        'SAMEORIGIN',
+      ],
+      'x-permitted-cross-domain-policies': [
+        'none',
+      ],
+      'x-xss-protection': [
+        '0',
+      ],
+    });
+    expect(response.status).toBe(204);
+    expect(response.type).toBe(undefined); // not specified
+    expect(await response.text()).toBe('');
+  });
+
+  test('Request: /_/report/errors responds with status 415', async () => {
+    const response = await fetch(`${fetchUrl}/_/report/errors`, {
+      ...defaultFetchOptions,
+      method: 'POST',
+      headers: {
+        origin: 'test.example.com',
+      },
+    });
+
+    expect(response.headers.raw()).toEqual({
```

Comment:
Do we need all the headers for these tests or can we use `expect.objectContaining` to target the most relevant ones?

Reply:
we could, I thought the feedback about using inline snapshot (https://github.com/americanexpress/one-app/pull/802#discussion_r938331534) was about checking all the headers beside readability. In my opinion, I think it's good to check all the them, especially if a new one gets added in the future, the `toEqual` assertion will enforce the check of all headers including the new ones